### ISSUE TITLE
Gallery Plugin Missing

### ISSUE THREAD

**Issue by patgmiller** [jrief/djangocms-cascade#102]
I am curious as to why the `GalleryPlugin` is not included in the `bootstrap3/settings.py` `CASCADE_PLUGINS`?

[cmsplugin_cascade/bootstrap3/settings.py#L7](https://github.com/jrief/djangocms-cascade/blob/master/cmsplugin_cascade/bootstrap3/settings.py#L7)

**Comment by jrief** [comment]
Because its not ready yet. And currently I don't need it urgently, so please be patient (... or diligent :)).
Jacob

**Comment by patgmiller** [comment]
If you have some direction you'd like to provide I'd like to contribute some of the work. I have a client site I am building with django-cms and `djangcms-cascade` covers about 90% of what I need. The gallery is the main part I am missing right now.

**Comment by jrief** [comment]
I have to reread the code. Its some time ago, that I started writing it.

Anyway, one thing I tried to do, is to create a generic inline plugin: ``InlineCascadeElement``. For Gallery it would have been the first of its kind, so this is an experimental approach. The reason for this approach is, that sometimes one wants to add a long list of items, say images (as for this gallery) or points of interest for a future Google/OSM-MapsPlugin. Here it would be tedious to use the parent-children approach as I did the Carousel- or AccordionPlugin. Instead I wanted to use the StackInlineAdmin to manage the children = images for this plugin in one editor window.
If this all works, it should be quite easy to adopt a template which renders the gallery. If you have other questions, try to contact me by email.
